balancer/grpclb/grpclb_test.go

```diff
@@ -411,15 +413,17 @@ func startBackendsAndRemoteLoadBalancer(numberOfBackends int, customUserAgent st
 	return
 }
 
-func (s) TestGRPCLB(t *testing.T) {
-	r := manual.NewBuilderWithScheme("whatever")
-
+// TestGRPCLB_Basic tests the basic case of a channel being configured with
+// grpclb as the load balancing policy.
+func (s) TestGRPCLB_Basic(t *testing.T) {
+	// Start a test backend and the remote balancer.
```

Comment:
This comment is redundant with the function name being called.  Consider instead a `t.Log` in `startBackendsAndRemoveLoadBalancer` (optional), or just delete.